mayastor/tests/nexus_multipath.rs

```diff
@@ -433,40 +440,43 @@ async fn nexus_resv_acquire() {
         .await
         .unwrap();
 
-    // Verify that it has grabbed the write exclusive reservation
+    // Verify that the second nexus has registered
     let v2 = get_nvme_resv_report(&rep_dev);
-    assert_eq!(v2["rtype"], 1, "should have write exclusive reservation");
-    assert_eq!(v2["regctl"], 1, "should have 1 registered controller");
+    assert_eq!(
+        v2["rtype"], 5,
+        "should have write exclusive, all registrants reservation"
+    );
+    assert_eq!(v2["regctl"], 2, "should have 2 registered controllers");
```

Comment:
If possible it'd be nice to define an actual type with human friendly fields, or maybe some friendly getter methods, though it may be more work than it's worth since this is only a test.

Reply:
I'll leave it as-is, it's only a test and it's still pretty readable.